components/ws-manager-mk2/controllers/timeout_controller.go

```diff
@@ -243,6 +243,7 @@ func (r *TimeoutReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	}
 
 	return ctrl.NewControllerManagedBy(mgr).
+		Named("timeout").
```

Comment:
the name shows up in the controller's metrics, defaulted to `workspace`, give it a unique name to differentiate the timeout controller metrics from the workspace controller